doc/user/content/serve-results/power-bi.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Power BI"
+description: "How to create dashboards with Power BI"
+aliases:
+  - /third-party/power-bi/
+  - /integrations/power-bi/
+menu:
+  main:
+    parent: "bi-tools"
+    name: "Power BI"
+    weight: 10
+---
+
+You can use [Power BI](https://powerbi.microsoft.com/) to create dashboards
+based on the data maintained in Materialize.
+
+## Database connection details
+
+To set up a connection from Power BI to Materialize, use the native
+[PostgreSQL database driver](https://learn.microsoft.com/en-us/power-query/connectors/postgresql#connect-to-a-postgresql-database-from-power-query-desktop)
+with the following parameters:
+
+Field                  | Value
+---------------------- | ----------------
+Database type          | **PostgreSQL database**
+Server                 | Materialize host name followed by `:6875`
```

Comment:
Does it make sense to add a concrete example here, like `id.us-east-1.aws.materialize.cloud:6875`?

Reply:
Added this as an example